c7n/policy.py

```diff
@@ -201,24 +201,25 @@ def run(self, *args, **kw):
             return
 
         with self.policy.ctx:
+            region        = (self.policy.options.region or 'default').ljust(14, ' ')
+            resource      = self.policy.resource_type.ljust(14, ' ')
+            start         = time.time()
+            resources     = self.policy.resource_manager.resources()
+            run_time      = time.time() - start
+            qty_resources = str(len(resources)).ljust(4, ' ')
             self.policy.log.info(
-                "Running policy %s resource: %s region:%s c7n:%s",
-                self.policy.name, self.policy.resource_type,
-                self.policy.options.region or 'default',
-                version)
-
-            s = time.time()
-            resources = self.policy.resource_manager.resources()
-            rt = time.time() - s
-            self.policy.log.info(
-                "policy: %s resource:%s has count:%d time:%0.2f" % (
-                    self.policy.name,
-                    self.policy.resource_type,
-                    len(resources), rt))
+                "region: %s resource: %s has count: %s time: %0.2f policy: %s" % (
+                    region,
+                    resource,
+                    qty_resources,
+                    run_time,
+                    self.policy.name
```

Comment:
self.policy.name is last, since it's length is arbitrary (sometimes quite long, like 50 chars). so if we keep this last, all the other shell output columns will align in a neatly human readable way.